CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+*              @developers
```

Comment:
Unfortunatelly, my username is `bedrich-schindler`.

Reply:
Oh, my bad 🙈